letsdrill.rb

```diff
@@ -1,24 +1,32 @@
 def get_letter_grade(integer)
- 
+ if integer >= 90
+   return "A"
+ elsif integer >= 80
+   return "B"
+ elsif integer >= 70
+   return "C"
+ elsif integer >= 60
+   return "D"
+ elsif integer >= 50
+   return "F"
   #Put your code here!
-
+  end
 end
 
 def shortest_string(array)
-
   #Put your code here!
-
+  array.sort { |x,y| x.length <=> y.length }.shift
```

Comment:
`shift` has a side effect of modifying the underlying Array (not that it matters here). `first` doesn't do that and is clearer to the next reader.